site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,117 @@
-alt="Context Menu Options page", width="357", height="222" %}
+The manifest is the configuration file of a Chrome extension. It is a required JSON file that must be located at the root of the project. It provides the browser with a blueprint of the extension, with important information such as:
 
-  <figcaption>
-    Options page in the extension's context menu.
-  </figcaption>
-</figure>
+- The name of the extension, a description of what it does, the current version number, and what icons to use.
+- The [Chrome API][api-ref] keys and [permissions][doc-perms] that the extension needs.
+- The files assigned as the extension service worker, the popup HTML file, the options page, the content scripts, etc.
 
-{% endColumn %}
+The [Manifest keys][doc-manifest] article contains the complete list of default and optional keys. For copy-paste-ready code samples, check out the [Manifest examples][doc-manifest-examples].
 
-{% endColumns %}
+### The service worker {: #background_script }
 
-See [Give users options][docs-options] to learn more.
+A service worker is an event-based script that the browser runs in the background. It is often used to process data, coordinate tasks in different parts of an extension, and as an extension's event manager. For example, the service worker can listen for and react to events when the extension is first installed, a new tab is created, a new bookmark is added, the extension toolbar icon is clicked, etc.
 
-### Additional HTML files {: #html-files}
+A service worker can access all the [Extension APIs][api-ref], but as a type of [Worker][mdn-worker] it can't use the typical DOM APIs that a document's global Window object provides. It also runs in its own environment, so it cannot directly modify a web page's content.
 
-An extension can also have other HTML files that are not declared in the manifest. All extension HTML files can access the [Chrome APIs][section-apis] and can use script tags including Javascript files, but cannot declare inline JavaScript.
+See [Handling events in the extension service worker][doc-sw] for more details. 
 
-You can open these pages using the web api [window.open()][mdn-window-open], the Chrome APIs
-[windows.create()][api-window-create], or [tabs.create()][api-create-tab].
+### Content scripts {: #content-scripts }
 
-## Extension files {: #files }
+Extensions use content scripts to inject code into host pages. They allow the extension to interact with and modify pages in the browser. For example, they can insert a new element on the page, change the style of a website, modify the [DOM][mdn-dom] elements, etc. 
 
-### Referencing extension files {: #ref-files }
+Content Scripts share access to the same DOM tree as the host page but run in a separate JavaScript environment. They also have access to a limited number of [Chrome APIs][api-ref]. See [Understanding content scripts][doc-content-scripts] for more details.
```

Comment:
Minor change to call out & link to a key terms. Please replace the inline link with a footer link.

```suggestion
Content Scripts share access to the same DOM tree as the host page but run in a separate JavaScript environment (the extension's [isolated world](/docs/extensions/mv3/content_scripts/#isolated_world)). They also have access to a limited number of [Chrome APIs][api-ref]. See [Understanding content scripts][doc-content-scripts] for more details.
```